manifests/init.pp

```diff
@@ -325,9 +329,11 @@
       register_in_foreman   => $register_in_foreman,
       foreman_base_url      => $foreman_url,
       registered_proxy_url  => "https://${capsule_fqdn}:${capsule::foreman_proxy_port}",
+      trusted_hosts         => undef,
```

Comment:
You want a separate PR for `trusted_hosts => undef` or something? It's not really related to #9102, it's a part of enabling the templates plugin.

Someday if 9102 is taken, we need a PR here to enable trusted hosts to be `[$::capsule_fqdn, $::parent_fqdn]` and enable some future `allow_all` to true on the templates, but that is not even on the horizon for 1.8.